public/themes/bootswatch/css/flatly.css

```diff
@@ -123,6 +123,9 @@ sub {
 
 img {
   border: 0;
+  border-radius: 20px;
```

Comment:
Jeneva, great job with the changes, I just had a few small things to mention before merging. First thing, can you re-name your commit? We're not supposed to put our names in our commits, it should just be descriptive of the change that is being made. Your username will still be tied to the commit of course.

Second, I think the border-radius change is having some unintended consequences on the home page graphics, or was that intentional? If intentional, maybe we can make the angle a bit more subtle (10-15px maybe?). The FAQ page looks much nicer, nice job.

Reply:
I question that we should be changing the 
Bootstrap file at all. I think that changing the bootstrap file might have consequences if we need to update bootswatch down the road.  I'd suggest using an override vile ... read answer #126 in this stack overflow article: https://stackoverflow.com/questions/8596794/customizing-bootstrap-css-template

Reply:
There are a number of issues with this pull request, and I have reached out to @jenevascherr on slack with the details. I will work with her to implement these changes in a better way.